docs/PacketFence_Clustering_Guide.asciidoc

```diff
@@ -897,27 +194,48 @@ type=internal
 mask=255.255.255.0
 
 ----
-Once this configuration is done, restart PacketFence to have it applied:
-  
- # systemctl restart packetfence  
+----
 
-If no error is found in the previous configuration, the previous restart of packetfence should have started two additional services : haproxy and keepalived
+[pf3.example.com]
+management_ip=192.168.1.7
 
-You should now have service on the IP addresses defined in the 'CLUSTER' sections
+[pf3.example.com interface eth0]
+ip=192.168.1.7
+type=management,high-availability
+mask=255.255.255.0
 
-Step 4 : Connect a slave packetfence server
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+[pf3.example.com interface eth1.2]
+enforcement=vlan
+ip=192.168.2.7
+type=internal
+mask=255.255.255.0
 
-First, connect the server to the database cluster using the instructions above
+[pf3.example.com interface eth1.3]
+enforcement=vlan
+ip=192.168.3.7
+type=internal
+mask=255.255.255.0
 
-On CentOS, add the following line in /etc/sysctl.conf and then reload with 'sysctl -p'
+----
+Once this configuration is done, reload the configuration and restart PacketFence to have it applied:
+  
+ # /usr/local/pf/bin/pfcmd configreload hard
+ # /usr/local/pf/bin/pfcmd checkup
+ # /usr/local/pf/bin/pfcmd service pf restart
 
-  net.ipv4.ip_nonlocal_bind = 1
+!!!!!!!!!!!!!!! INSTRUCTIONS ON MODIFYING THE TARGET FOR SYSTEMD HERE....
```

Comment:
Not even that.
All that does is make sure the service is enabled in systemd, but since mariadb must be enabled in standalone as well that should be unnecessary.